ClientApp/viking-vault/src/UserDashboard/TransactionList.tsx

```diff
@@ -47,16 +50,16 @@ class TransactionList extends React.Component<any, IState> {
     }
 
     private formatOtherPartyString(transaction: ITransaction) {
-        if(transaction.type === "transfer") {
+        if(transaction.type.toLowerCase() === "transfer") {
```

Comment:
Why didn`t you also create an enum for transactionType? As a rule of tumb, we don`t just put hardcoded strings through code.